### PR TITLE
Guaymas transect branch

### PR DESCRIPTION
**Pull request by vpreston**
This branch is the main development branch used to examine the Guaymas basin transect mission. A version of this code is frozen and tagged during the review process. This PR is primarily to update the main development repository, as several basic tools were updated or developed that may be useful for other analyses.

### CODE REVIEW

File: guaymas_data_analysis/utils/data_utils.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def convert_oceanographic_measurements(df: pd.DataFrame, vehicle="sentry"):
     # calculate practical salinity
     df.loc[:, "practical_salinity"] = df.apply(
         lambda x: gsw.SP_from_C(
-            C=x["ctd_cond"],
+            C=x["ctd_cond"]*10,
```
Comment (geflaspohler) [inline guaymas_data_analysis/utils/data_utils.py:55]: Would be good to add a comment here for why the multiplication by 10 is necessary

File: guaymas_missions/sentry_rosette_transect/transect_data_parsing.py [hunk 2]
```diff
@@ -109,22 +110,34 @@
     bott_df["bottle_pos"] = bott_df["bottle_pos"].astype(np.int32)
     gga_df = pd.read_table(INPUT_GGA, delimiter=",")
     gga_df["CTD Bottle #"] = gga_df["CTD Bottle #"].astype(np.int32)
+    
     gga_df.loc[:, 'ch4_ppm_corr_05'] = gga_df["GGA Methane"] / 0.05
     gga_df.loc[:, 'ch4_ppm_corr_15'] = gga_df["GGA Methane"] / 0.15
+    gga_df.loc[:, 'ch4_ppm_corr_12'] = gga_df["GGA Methane"] / 0.12
```
Comment (geflaspohler) [inline guaymas_missions/sentry_rosette_transect/transect_data_parsing.py:116]: Commenting the source and reason for these corrections would help us in the future; probably worth doing

**Review by geflaspohler — APPROVED** [review]
I didn't go through all of the new code files in detail for correctness - I can do this if you'd like? But the structure/API changes look good. 

My biggest comment was that we should add some more commenting when we make corrections/multiply by constants/etc., so that we can remember why that was the rigth thing a few years from now. 

Otherwise, happy to merge this new functionality in :)

File: guaymas_missions/sentry_rosette_transect/transect_data_parsing.py (hunk 2 above)
Comment (vpreston) [reply guaymas_missions/sentry_rosette_transect/transect_data_parsing.py:119]: Good idea! Added!

File: guaymas_data_analysis/utils/data_utils.py (hunk 1 above)
Comment (vpreston) [reply guaymas_data_analysis/utils/data_utils.py]: Good catch -- this is just a unit conversion from S/m to mS/cm as per the GSW library input guidelines. Added a comment!

**Pull request merged**